docs/appendix/zowe-glossary.md

```diff
@@ -1,6 +1,168 @@
 # Glossary
 
-This glossary is part of a growing list of Zowe terminology.
+This glossary is part of a growing list of Zowe terminology # Glossary of Zowe terminology and includes both technical as well as organizational terms that are specific to Zowe.
+
+## Main Components
```

Comment:
Be careful, components are very low level concepts and what you are stating below applies for the project level. 

For example API Mediation Layer isn't a component. It is project that consists of multiple components e.g. API Gateway, API Catalog, Caching service.